### PR TITLE
[CES-546] Added Virtual Machines for ActiveGate and OneAgent

### PR DESCRIPTION
**Pull request by mamu0**
Added module for Virtual Machine and a configuration file for all resources, needed Dynatrace token

### CODE REVIEW

File: infra/resources/dev/templates/config.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+
+processors:
```
Comment (gunzip) [inline infra/resources/dev/templates/config.yaml:6]: can you add a link to documentation?

File: infra/resources/dev/templates/config.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      - key: collector.hostname
+        value: $HOSTNAME
+        action: insert
+      - key: azure.container_app.revision
+        value: $CONTAINER_APP_REVISION
+        action: insert
+      - key: azure.container_app.name
+        value: $CONTAINER_APP_NAME
+        action: insert
+      - key: source.blog
```
Comment (gunzip) [inline infra/resources/dev/templates/config.yaml:19]: what's blog in this context?

File: infra/resources/dev/templates/config.yaml (hunk 1 above)
Comment (mamu0) [reply infra/resources/dev/templates/config.yaml]: sorry I added this folder for a test, but it's not needed 😆  now I'll delete it

File: infra/resources/dev/templates/config.yaml (hunk 2 above)
Comment (mamu0) [reply infra/resources/dev/templates/config.yaml]: ditto

**Review by gunzip — APPROVED** [review]

**Pull request merged**